src/Randomizer.App/RomGenerator.cs

```diff
@@ -90,24 +94,29 @@ public bool GenerateRandomRom(RandomizerOptions options, out string path, out st
                         "Retrying to generate the seed may work, but the selected settings may be impossible to generate successfully and will need to be updated.\n" +
                         "Continue with the current seed that does not meet all requested settings?", "SMZ3 Cas’ Randomizer", MessageBoxButtons.YesNo) == DialogResult.No)
                 {
-                    path = null;
-                    error = "";
-                    rom = null;
-                    return false;
+                    return null;
                 }
                 else
                 {
-                    error = "";
-                    rom = GenerateRomInternal(seed, options, out path, out error);
-                    return true;
+                    var results = await GenerateRomInternalAsync(seed, options);
+                    if (!string.IsNullOrEmpty(results.MsuError))
+                    {
+                        System.Windows.Forms.MessageBox.Show("There was an error assigning the MSU.\n" + results.MsuError, "SMZ3 Cas’ Randomizer", MessageBoxButtons.OK, MessageBoxIcon.Warning);
```

Comment:
This error message is pretty cryptic. Maybe we can make this a little clearer?

Reply:
This one merely outputs the MSU generation error, which currently is set to: 

error = $"Due to technical limitations, the MSU-1 " +
                    $"pack and the ROM need to be on the same drive. MSU-1 " +
                    $"support cannot be enabled.\n\nPlease move or copy the MSU-1 " +
                    $"files to somewhere on {romDrive}, or change the ROM output " +
                    $"folder setting to be on the {msuDrive} drive.";

That being said, I don't think a "There was an error assigning the MSU" intro is really necessary as the actual error message being returned by MSU generation is pretty self explanatory, so I'll change this to simply output the MSU generation error by itself.